### ISSUE TITLE
Slightly longer time to screencap for slower pages?

### ISSUE THREAD

**Issue by encaitor** [conceptualspace/yet-another-speed-dial#64]
Hi,

Is there a possibility to add an option for some more time to grab a screencap for the thumbnail?

E.g this site has a bunch of pictures and it seems to not load long enough before capturing to grab them all.
![bild](https://user-images.githubusercontent.com/19467159/171905292-22ec84cb-8c9e-4c8f-91c2-57f2d974c658.png)

**Comment by conceptualspace** [comment]
good idea. for now if you want to get the complete image you can open the site first and then bookmark it to the speed dial folder or right click on the page and select 'add to speed dial' -- that way the page is already loaded when YASD generates the screenshot

**Comment by encaitor** [comment]
Yeah, that's what I did to make it work as intended. Just migrated from Opera tho so having to do it for 30+ bookmarks was slightly annoying :D Figured I'd at least mention it as an improvement.
